docs/DeveloperGuide.md

```diff
@@ -543,30 +543,55 @@ The application provides users with two different methods of entering or editing
 * Using the pop-up text box implemented in this feature
 * Adding the remark through the [add feature](#add-feature)
 
+The remark mechanism will be facilitated by a pop-up text box.
+They can do this by simply omitting the optional `REMARK` parameter, which will cause the pop-up text box to appear.
+This will allow users to format their remarks however they like, rather than being restricted to a single line in the command line.
+Users can use the `CTRL` + `S` combination to close the remark box without using the mouse to click on the "X", thereby
+staying through to the goal of catering to fast typists.
+
+However, should the user prefer it, they can also directly input their remarks into the pop-up text box
+by specifying the optional `REMARK` parameter.
+
+Currently, to differentiate between an empty remark specified from the text box and the `remark INDEX` command to open
```

Comment:
U'll need to first explain how the remark command words in the back end. How it uses itself to enter the text from the popup.

idk if u added it above, cuz im my phone cant rlly see